ui/raidboss/raidboss_config.js

```diff
@@ -554,7 +554,10 @@ class RaidbossConfigurator {
           div.appendChild(input);
           input.type = 'text';
           input.step = 'any';
-          input.placeholder = this.base.translate(kMiscTranslations.valueDefault);
+          if (typeof trig.durationSeconds === 'number')
+            input.placeholder = `${trig.durationSeconds} ${this.base.translate(kMiscTranslations.valueDefault)}`;
```

Comment:
```suggestion
            input.placeholder = `${trig.durationSeconds}`;
```

What about just showing the number? The placeholder should be a different color.

Reply:
Oh, this is @MaikoTan suggests before. I think a single number only is a little empty in box.

Reply:
I think it looks ok (although the CSS change makes it look emptier)?

![image](https://user-images.githubusercontent.com/462317/122868243-8ec8db00-d2df-11eb-844c-a376da3d0a6d.png)

Reply:
Fine. This css change is actually for de, for I find that default in de is a little longer to show.